integrations/June/browser.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable */
+import logger from "../../utils/logUtil";
+import get from "get-value";
+import { NAME } from "./constants";
+import { LOAD_ORIGIN } from "../ScriptLoader";
+import { getDestinationExternalID } from "../utils/commonUtils";
+
+class June {
+  constructor(config) {
+    this.name = NAME;
+    this.apiKey = config.apiKey;
+  }
+
+  loadScript() {
+    window.analytics = {};
+    function juneify(writeKey) {
+      window.analytics._writeKey = writeKey;
+      const script = document.createElement("script");
+      script.type = "application/javascript";
+      script.src =
+        "https://unpkg.com/@june-so/analytics-next/dist/umd/standalone.js";
+      script.setAttribute("data-loader", LOAD_ORIGIN);
+      const first = document.getElementsByTagName("script")[0];
+      first.parentNode.insertBefore(script, first);
+    }
+    juneify(this.apiKey);
+  }
+
+  init() {
+    logger.debug("===In init June===");
+    this.loadScript();
+  }
+
+  isLoaded() {
+    logger.debug("===In isLoaded June===");
+    return !!window.analytics && typeof window.analytics === "object";
+  }
+
+  isReady() {
+    logger.debug("===In isReady June===");
+    return !!window.analytics;
```

Comment:
This check should be same as `isLoaded` unless there is any ready API.

Reply:
updated